testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -159,24 +162,43 @@ Feature: Smoke tests for <client>
     And I click on "Expand All Sections"
     And I should see a "Enable and configure Prometheus exporters for managed systems." text
     And I check "node" exporter
```

Comment:
```suggestion
    And I check "node" exporter
    And I click on "Save"
    Then I should see a "Formula saved" text
```

Reply:
you're missing saving the formula before checking if it's been saved

Reply:
Nice catch. Thank you!

Reply:
Fixed.